platform_versioned_docs/version-23.4.0/data/data-studios.mdx

```diff
@@ -180,6 +180,8 @@ When you start a data studio, it automatically creates a *checkpoint*. A checkpo
 
 When you restart a data studio, or start a new data studio from a previously created checkpoint, changes such as installed software packages and configuration files are restored and made available in the data studio session. Changes made to mounted datasets are not included in a checkpoint.
 
+Checkpoint files in the compute environment work directory may be shared by multiple studios. Each checkpoint file is cleaned up asynchronously by Seqera Platform after the last studio referencing the checkpoint is deleted from the platform. The cleanup process is best effort. Seqera Platform tries to remove the checkpoint, but it can fail, such as when the compute environment credentials used do not have sufficient permissions to delete objects from storage buckets.
```

Comment:
```suggestion
Checkpoint files in the compute environment work directory may be shared by multiple data studios. Each checkpoint file is cleaned up asynchronously by Seqera Platform after the last data studio referencing the checkpoint is deleted.

:::note
The cleanup process is best effort and not guaranteed. Seqera Platform tries to remove the checkpoint, but it can fail. For example, if the compute environment credentials used do not have sufficient permissions to delete objects from storage buckets.
:::
```